base/Dockerfile

```diff
@@ -47,6 +50,17 @@ RUN apt-get update && \
         curl -s -S -o - "${KOPANO_CORE_REPOSITORY_URL}/Release.key" | apt-key add -; \
     fi
 
+# Create kopano user and group
+RUN groupadd -r --gid ${KOPANO_GID} kopano
+RUN useradd -r -s /usr/sbin/nologin -d /var/lib/kopano -g ${KOPANO_GID} -u ${KOPANO_UID} kopano
+
+# Fix system groups. Otherwise some packages might not install correctly (for example kopano-kapid requires the kopano group 
```

Comment:
this block should no longer be necessary, now that user and group are system group

Reply:
But what if someone sets UID or GID for the kopano user to a number >= 1000 ? Then kapi's installation script will fail without this workaround.

Reply:
Ok, so the error message I see is the following:

```
install: invalid user 'kapi'
dpkg: error processing package kopano-kapid (--configure):
 subprocess installed post-installation script returned error exit status 1
```

This was with uid/gid set to 1234

Reply:
ok, I think I've understood it now. when you call `addgroup` with with a uid >999 then regardless if you passed --system it will not be a system group. which then leads to the && chain in the kapi postinst to exit before it created the kapi user.

I have discussed this internally a bit and the general sentiment to modifying `adduser.conf` is "don't do that!"

So I'd rather have a note saying that only uid/gid below up until 999 are allowed than trying to work around this limit.

Reply:
Yeah it is indeed a dirty workaround, I will rework/remove the section in the Dockerfile. Are there any chances that this gets fixed in the kapi installation script?

Reply:
> Are there any chances that this gets fixed in the kapi installation script?

considering that this only breaks if there is a pre-existing group with a gid >999 there is no real need to fix this.

Reply:
> > Are there any chances that this gets fixed in the kapi installation script?
> 
> considering that this only breaks if there is a pre-existing group with a gid >999 there is no real need to fix this.

This might lead to confusion in future as it is inconsistent with the kopano-common package, where a working check for a already existing group is implemented. Also, what if a customer with an old (maybe a non-dockerized) Kopano installation tries to install kapi with a already existing, non-system group?  But maybe this should be discussed elsewhere, it is kinda offtopic ;)